.travis/klee.sh

```diff
@@ -51,9 +51,9 @@ fi
 ###############################################################################
 # Handle setting up solver configure flags for KLEE
 ###############################################################################
-KLEE_Z3_CONFIGURE_OPTION=""
-KLEE_STP_CONFIGURE_OPTION=""
-KLEE_METASMT_CONFIGURE_OPTION=""
+KLEE_Z3_CONFIGURE_OPTION="-DENABLE_SOLVER_Z3=OFF"
```

Comment:
I know, we should remove it, but you probably forgot that we still support Autoconf..